CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 * **Interface Update!** `PolymerProject.analyzer` is no longer a required step in your build pipeline. Instead, analysis happens automatically while it fills the `project.sources()` and `project.dependencies()` streams with your project files. See [the README](/README.md) for updated examples of what build streams look like without the analyzer.
 * `StreamAnalyzer` is now `BuildAnalyzer` (since it is no longer a stream). This change only affects uses who are importing and/or using `StreamAnalyzer` directly from the `polymer-build` module.
+* **[`merge-stream`](https://github.com/grncdr/merge-stream) users:** Update to v1.0.1 or later if you are using `merge-stream` with `poylmer-build`. Stream errors do not propagate properly in previous versions of the library, and your build task may silently fail as a result.
```

Comment:
In polymer-build, we print all warnings and then emit a single error if any of them were of type "ERROR" (https://github.com/Polymer/polymer-build/blob/536c7a846d1f854a7f244695c001af0bc853573a/src/analyzer.ts#L258)